### PR TITLE
feat: fix the icon in conversation list - ACC - 263

### PR DESCRIPTION
**Pull request by BrzozowskiPawel**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Fix the icon in the top right corner that directs to the settings.

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by BrzozowskiPawel** [comment]
This PR is not ready to be checked yet. I need to discuss a couple of things before final changes.

**Review by agisilaos — APPROVED** [review]

File: Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift [hunk 1]
```diff
@@ -128,7 +128,7 @@ class BaseAccountView: UIView {
         case .current:
             return account.unreadConversationCount > 0
         case .others:
-            return ((SessionManager.shared?.accountManager.totalUnreadCount ?? 0) - account.unreadConversationCount) > 0
+            return (SessionManager.shared?.accountManager.totalUnreadCount ?? 0) > 0
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift:131]: Why did you change this?

**Comment by KaterinaWire** [comment]
@BrzozowskiPawel Could you add more description what kind of the changes are in this PR?

**Comment by BrzozowskiPawel** [comment]
After some chatting with Anette and Rahil they asked me to change functionality a bit so I will make it a draft again.

File: Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift (hunk 1 above)
Comment (BrzozowskiPawel) [reply Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift:131]: **Reverted.**

I was trying to change it to match desired behaviour but after the talk, I have had with Anette and Rahil it's now outdated and I will revert it.

**Comment by BrzozowskiPawel** [comment]
@KaterinaWire @agisilaos 
Fixed.

File: Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift [hunk 2]
```diff
@@ -216,6 +216,12 @@ class BaseAccountView: UIView {
         updateAppearance()
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
```
[the remark this answers is not in the document]
Comment (BrzozowskiPawel) [reply Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift:221]: Sounds good! Thanks for this feedback. I have read more about it in the documentation and you are right!

**Comment by BrzozowskiPawel** [comment]
To be clear:
- the "border" is now visible always.
- the dot icon indicates an unread message only for the account that has an unread message.

**Review by KaterinaWire — APPROVED** [review]
